etc/dehydrated/hook.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
I didn't think `set -e` worked that way for bash function calls. I could be mistaken. This script defines a handful of functions. Then `dehydrated` sources this file and invokes those functions at the proper time. If `set -e` would work in this context, I'm not sure what impact it would have on `dehydrated`. Perhaps the better approach here would be to `&&` the `curl` commands? Something like:

```bash
curl blah1 &&
curl blah2 &&
curl blah3
```

Thoughts?